deepchem/data/dft_data.py

```diff
@@ -0,0 +1,319 @@
+"""
+Density Functional Theory Data
+Derived from: https://github.com/mfkasim1/xcnn/blob/f2cb9777da2961ac553f256ecdcca3e314a538ca/xcdnn2/entry.py
+"""
+from __future__ import annotations
+import os
+from abc import abstractmethod, abstractproperty
+from typing import List, Dict, Optional, Union
+import numpy as np
+import torch
+try:
+    import dqc
+    from dqc.system.mol import Mol
+    from dqc.system.base_system import BaseSystem
+    from dqc.grid.base_grid import BaseGrid
+    from deepchem.utils.dftutils import KSCalc
+except ModuleNotFoundError:
+    raise ModuleNotFoundError("This utility requires dqc")
+
+import yaml
+from yaml.loader import SafeLoader
+
+
+class DFTSystem(dict):
```

Comment:
Can you add these new classes to the docs/ folder in the .rst files for the data classes?